Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch

janeyx99 opened the pull request:
Passing CI: https://github.com/pytorch/pytorch/actions/runs/18141589975/job/51635340255?pr=163782

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #163782

janeyx99 committed:
Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch

[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

huydhn reviewed (approved):
LGTM!

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"




[ghstack-poisoned]

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"

Passing CI: https://github.com/pytorch/pytorch/actions/runs/18138533766/job/51624876117?pr=163782




[ghstack-poisoned]

malfet commented on .github/workflows/_linux-test-stable-fa3.yml:
```diff
@@ -0,0 +1,256 @@
+# The point of this workflow is to test that a FA3 wheel that was built based off the
```
Why not do it inside `test.sh`? I.e. just call your shard `test-fa3` but rely on the same .yml file?

mikaylagawarecki reviewed (approved):
test_fa3_abi_stable.sh looks good to me

janeyx99 committed:
Update on "Add smoke tests to verify that stable ABI FA3 wheel runs w/ newer torch"

Passing CI: https://github.com/pytorch/pytorch/actions/runs/18140599635/job/51631885122?pr=163782




[ghstack-poisoned]

janeyx99 commented:
@pytorchbot merge
